### ISSUE TITLE
Trying this in windows

### ISSUE THREAD

**Issue by MXP2095onetechguy** [jdah/minecraft-weekend#47]
I am seeing if this works in windows by building it in windows
trying wsl and then native windows

**Comment by MXP2095onetechguy** [comment]
No, WSL seems to have problem downloading one of the packages and msys2's(that is what i use to make the native version) make have problems with the makefile.

**Comment by MXP2095onetechguy** [comment]
trying on msys2 again, but modified makefile

**Comment by CrezyDud** [comment]
hahaha, windows users

**Comment by UniqueName12345** [comment]
> hahaha, windows users

Please don't be offensive to other people on github. Thank you!

**Comment by legendaryzyper** [comment]
I did manage to get it working on Visual Studio with some changes to compiler-specific hacks.

**Comment by ghost** [comment]
> > hahaha, windows users
> 
> Please don't be offensive to other people on github. Thank you!

nah man, dont practive coorparate speech (thank you!)
